### PR TITLE
Update description of a database info reply

### PR DESCRIPTION
**Pull request by eiri**
## Overview

This adds description of a new information block `cluster` and updates description of `update_seq`.

## Testing recommendations

Run linter with  `make check`

## Related Pull Requests

Related to apache/couchdb#837 and #164

## Checklist

- [x] Documentation is written and is accurate;
- [x] `make check` passes with no errors
- [ ] Update [rebar.config.script](https://github.com/apache/couchdb/blob/master/rebar.config.script) with the commit hash once this PR is rebased and merged

### CODE REVIEW

File: src/api/database/common.rst [hunk 1]
```diff
@@ -55,6 +55,12 @@
                      - :mimetype:`text/plain`
     :>header Content-Type: - :mimetype:`application/json`
                            - :mimetype:`text/plain; charset=utf-8`
+    :>json number cluster.n: Replicas. The number of copies of every document.
+    :>json number cluster.q: Shards. The number of range partitions.
+    :>json number cluster.r: Read quorum. The number of consistent copies
+      of a document that need to be read before successfull reply.
```
Comment (flimzy) [inline src/api/database/common.rst]: successfull -> successful

File: src/api/database/common.rst [hunk 2]
```diff
@@ -55,6 +55,12 @@
...
+    :>json number cluster.w: Write quorum. The number of copies of a document
+      need to be written before successfull reply.
```
Comment (flimzy) [inline src/api/database/common.rst]: successfull -> successful

Comment (flimzy) [reply src/api/database/common.rst]: Maybe more clear "The number of copies of a document that need to be written before successful reply"?

**Comment by flimzy** [comment]
+1 looks good to me in terms of formatting. I can't vouch for its technical accuracy, though.

File: src/api/database/common.rst [hunk 3]
```diff
@@ -55,6 +55,12 @@
                      - :mimetype:`text/plain`
     :>header Content-Type: - :mimetype:`application/json`
                            - :mimetype:`text/plain; charset=utf-8`
+    :>json number cluster.n: Replicas. The number of copies of every document.
+    :>json number cluster.q: Shards. The number of range partitions.
+    :>json number cluster.r: Read quorum. The number of consistent copies
+      of a document that need to be read before successful reply.
```
Comment (nickva) [inline src/api/database/common.rst]: a successful reply

File: src/api/database/common.rst [hunk 4]
```diff
@@ -55,6 +55,12 @@
...
+    :>json number cluster.w: Write quorum. The number of copies of a document
+      that need to be written before successful reply.
```
Comment (nickva) [inline src/api/database/common.rst]: a successful reply

File: src/api/database/common.rst [hunk 5]
```diff
@@ -75,7 +81,9 @@
       in bytes.
     :>json number sizes.file: The size of the database file on disk in bytes.
       Views indexes are not included in the calculation.
-    :>json number update_seq: The current number of updates to the database.
+    :>json string update_seq: An opaque string that describes the state
+      of the database. Do not rely on this string for counting the number
+      of updates.
```
Comment (nickva) [inline src/api/database/common.rst]: Good point on mentioning not relying on the format for counting updates.

**Comment by nickva** [comment]
+1 (but don't forget to squash)

**Pull request merged**